src/ol_infrastructure/infrastructure/aws/kms/__main__.py

```diff
@@ -0,0 +1,84 @@
+"""State to create symmetric AWS KMS Customer Master Keys (CMK) for use
+with EC2 volume encryption. 
+"""
+import json
+from pulumi import get_stack
+from pulumi_aws import get_caller_identity, kms
+
+
+owner = str(get_caller_identity().account_id)
+stack = get_stack()
+stack_name = stack.split('.')[-1]
+env_suffix = stack_name.lower()
+kms_ec2_environment = f'ec2-ebs-{env_suffix}'
+
+kms_ec2_ebs_encryption_policy = {
+    'Version': '2012-10-17',
+    'Id': 'auto-ebs-2',
+    'Statement': [
+        {
+            'Sid': 'Allow access through EBS for all principals in the account that are authorized to use EBS',
+            'Effect': 'Allow',
+            'Principal': {
+                'AWS': '*'
+            },
+            'Action': [
+                'kms:Encrypt',
+                'kms:Decrypt',
+                'kms:ReEncrypt*',
+                'kms:GenerateDataKey*',
+                'kms:CreateGrant',
+                'kms:DescribeKey'
+            ],
+            'Resource': '*',
+            'Condition': {
+                'StringEquals': {
+                    'kms:ViaService': 'ec2.us-east-1.amazonaws.com',
```

Comment:
Not currently an issue, as we only use us-east-1 currently, but should we make this applicable to any ec2 region?